config/environments/production.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  
+  # Set the value below to false if you are terminating SSL upstream and proxying to HTTP:
```

Comment:
Regarding this comment, I think it's too specific to be permanently here. Better to create a separate issue and people with this issue can search these keywords.

Reply:
I agree. It was late when I was working on this.  After further review, this comment needs to go into the Dockerfile under the commented out production configuration.  I will update this.

Reply:
See new pull request #236 which removes the production.rb comment from the equation, but adds to the Dockerfile comments, because this is normally the way Docker apps are deployed in production.